docs/user-roles.rst

```diff
@@ -44,3 +57,63 @@ also perform every available operation inside that project.
     organization.
 
 A *project member* has read-only access to the project.
+
+
+.. _user-ops:
+
+User operations
```

Comment:
What about naming this section "Database users" or "CrateDB users" (vs. Cloud users)?